src/test/java/org/folio/rest/impl/PostGobiOrdersHelperTest.java

```diff
@@ -355,9 +361,54 @@ public final void testLookupOrderMappings(TestContext context) throws Exception
           } catch (Exception e) {
             logger.error("Failed to execute translation", e);
           }
+
+          context.assertNotNull((map.get(Field.ESTIMATED_PRICE)));
+          ds = map.get(Field.ESTIMATED_PRICE);
+          context.assertEquals("//NetPrice/Amount", ds.from);
+          context.assertNotNull(ds.defValue);
+          DataSource defVal = (DataSource) ds.defValue;
+          context.assertEquals("//ListPrice/Amount//EstPrice", defVal.from);
+          context.assertEquals("15.0", defVal.defValue);
+          try {
+            Double result = (Double) defVal.translation.apply(defVal.defValue.toString()).get();
+            context.assertEquals(15.0, result);
+          } catch (Exception e) {
+            logger.error("Failed to execute translation", e);
+          }
+
           vertx.close(context.asyncAssertSuccess());
           async.complete();
         });
     });
   }
+
+  @Test
+  public final void testextractOrderMappingWithOtherField(TestContext context) throws Exception {
```

Comment:
Fix method name case... testExtract

Reply:
Also, this test case probably doesn't belong in this test file since it doesn't pertain to PostGobiOrdersHelper.java.